msr-flows/msbr_compare/serpent_input.py

```diff
@@ -0,0 +1,375 @@
+import misc_funcs
+import user_input
+from jinja2 import Environment, FileSystemLoader
+import os
+from os import path
+from time import sleep
+
+
+class create_deck:
+    '''
+    This class is used to build the deck which will be fed into Serpent2
+
+    '''
+
+    def __init__(
+            self,
+            reprocessing,
+            read_file,
+            read_time,
+            write_file,
+            base_material_file,
+            template_name,
+            template_path,
+            step_days,
+            list_inventory,
+            identifier,
+            deck_name):
+        '''
+        Initialize the `create_deck` class. Files need proper path.
+
+        Parameters
+        ----------
+        reprocessing : dictionary
```

Comment:
You should add what the keys and values for the dictionary like this:

```
        reprocessing : dict
            Dictionary that...

            ``key``
                key description
            ``value``
                value description

```
do this for all `dict` parameters in your function docstrings.